src/anomalib/trainer/connectors/thresholding_connector.py

```diff
@@ -22,48 +20,29 @@ class ThresholdingConnector:
 
     Args:
         trainer (trainer.AnomalibTrainer): Trainer object
-        threshold_method (ThresholdMethod): Thresholding method to use. Defaults to ``ThresholdMethod.ADAPTIVE``.
-        manual_image_threshold (Optional[float]): Image threshold in case manual threshold is used. Defaults to None.
-        manual_pixel_threshold (Optional[float]) = Pixel threshold in case manual threshold is used. Defaults to None.
+        image_threshold_method (dice | None): Thresholding method. If None, adaptive thresholding is used.
+        pixel_threshold_method (dice | None): Thresholding method. If None, adaptive thresholding is used.
     """
 
     def __init__(
         self,
         trainer: "trainer.AnomalibTrainer",
-        threshold_method: ThresholdMethod = ThresholdMethod.ADAPTIVE,
-        manual_image_threshold: float | None = None,
-        manual_pixel_threshold: float | None = None,
+        image_threshold_method: dict | None = None,  # TODO change this in the CLI
+        pixel_threshold_method: dict | None = None,
     ) -> None:
-        if threshold_method == ThresholdMethod.ADAPTIVE and all(
-            i is not None for i in (manual_image_threshold, manual_pixel_threshold)
-        ):
-            warn(
-                "When `threshold_method` is set to `adaptive`, `manual_image_threshold` and `manual_pixel_threshold` "
-                "must not be set. Ignoring manual thresholds."
-            )
-
-        if threshold_method == ThresholdMethod.MANUAL and all(
-            i is None for i in (manual_image_threshold, manual_pixel_threshold)
-        ):
-            raise ValueError(
-                "When `threshold_method` is set to `manual`, `manual_image_threshold` and `manual_pixel_threshold` "
-                "must be set."
-            )
-
-        self.threshold_method = threshold_method
-        self.manual_image_threshold = manual_image_threshold
-        self.manual_pixel_threshold = manual_pixel_threshold
+        self.image_threshold_method = image_threshold_method
+        self.pixel_threshold_method = pixel_threshold_method
+        self.image_threshold: BaseAnomalyThreshold
+        self.pixel_threshold: BaseAnomalyThreshold
         self.trainer = trainer
 
     def initialize(self) -> None:
-        """Assigns pixel and image thresholds to the model.
-
-        This allows us to export the metrics along with the torch model.
-        """
-
-        if self.threshold_method == ThresholdMethod.MANUAL:
-            self.trainer.pixel_threshold.value = torch.tensor(self.manual_pixel_threshold).cpu()
-            self.trainer.image_threshold.value = torch.tensor(self.manual_image_threshold).cpu()
+        """Assigns pixel and image thresholds to the Anomalib trainer."""
```

Comment:
Is this docstring still accurate? Thresholds are no longer assigned to trainer right?